VirtoCommerce.CatalogModule.Web/Scripts/blades/property-dictionary-list.tpl.html

```diff
@@ -0,0 +1,25 @@
+<div class="blade-static">
+    <div class="form-group">
+        <div class="form-input" style="flex-grow: 1;">
+            <input ng-model="filter" ng-keyup="searchFilter(filter)" style="width: calc(100% - 80px);"
+                placeholder="{{'platform.placeholders.search-keyword' | translate}}">
+            <button class="btn __other" style="position: relative;right: 45px;" ng-click="filter = ''; searchFilter(filter)">
+                <i class="btn-ico fa fa-remove" title="Clear"></i>
+            </button>
+        </div>
+    </div>
+</div>
+<div class="blade-content">
+    <div class="blade-inner">
+        <div class="inner-block clearfix">
+            <div class="note" ng-if="!groupedDictionaryValues.length">{{ 'platform.list.no-data' | translate }}</div>
+            <ul class="list __items">
+                <li class="list-item" ng-repeat="property in groupedDictionaryValues track by $index" ng-class="{'__selected': property === setSelectedNode}"
+                    ng-click='selectNode(property)'>
+                    <span style="font-weight:bold; margin-right:12px;">{{property.alias}}</span>
```

Comment:
could we use css classes defined at the module level (content) instead of using inline styles?